languages/nodejs/repl.js

```diff
@@ -73,9 +73,13 @@ const isTTY = isatty(process.stdin.fd);
 // Red errors (if stdout is a TTY)
 function logError(msg) {
   if (isTTY) {
-    process.stdout.write(`\u001b[0m\u001b[31m${msg}\u001b[0m`);
+    process.stderr.write(`\u001b[0m\u001b[31m${msg}\u001b[0m`);
   } else {
-    process.stdout.write(msg);
+    process.stderr.write(msg);
+  }
+
+  if (!msg.endsWith('\n')) {
+    process.stderr.write('\n');
   }
 
   if (!msg.endsWith('\n')) {
```

Comment:
is this block still needed? IIUC this would print an extra newline (to stdout, no less!)

Reply:
idts, I think I just forgot to remove it.